tekton/api/golang-db-test.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: golang-db-test
```

Comment:
Cant, we use the catalog task for this?

Reply:
unit test requires db pod, so we have created a task for it as it is not there in catalog